mllib/src/main/scala/org/apache/spark/mllib/linalg/distributed/RowMatrix.scala

```diff
@@ -498,6 +498,65 @@ class RowMatrix(
   }
 
   /**
+   * Compute QR decomposition for rowMatrix. The implementation is designed to optimize the QR
+   * decomposition (factorizations) for the RowMatrix of a tall and skinny shape, yet it applies
+   * to RowMatrix in general.
```

Comment:
We can remove `, yet it applies`. It kinda encourages people to try it on fat matrices.